src/status_im2/contexts/activity_center/notification/community_request/view.cljs

```diff
@@ -0,0 +1,66 @@
+(ns status-im2.contexts.activity-center.notification.community-request.view
+  (:require [quo2.core :as quo]
+            [status-im2.constants :as constants]
+            [status-im2.contexts.activity-center.notification.common.style :as common-style]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [utils.datetime :as datetime]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- swipeable
+  [{:keys [active-swipeable extra-fn]} child]
+  [common/swipeable
+   {:left-button      common/swipe-button-read-or-unread
+    :left-on-press    common/swipe-on-press-toggle-read
+    :right-button     common/swipe-button-delete
+    :right-on-press   common/swipe-on-press-delete
+    :active-swipeable active-swipeable
+    :extra-fn         extra-fn}
+   child])
+
+(defn- get-header-text-and-context
+  [community membership-status timestamp]
+  (let [community-name        (:name community)
+        community-image       (get-in community [:images :thumbnail :uri])
+        community-context-tag [quo/context-tag common/tag-params {:uri community-image}
+                               community-name]
+        request-timed-out?    (> (datetime/timestamp)
+                                 (+ timestamp constants/activity-center-community-request-timeout-ms))]
+    (cond
+      (= membership-status constants/activity-center-membership-status-pending)
+      (if request-timed-out?
+        {:header-text (i18n/label :t/community-request-not-accepted)
+         :context     [[quo/text {:style common-style/user-avatar-tag-text}
+                        (i18n/label :t/community-request-not-accepted-body-text-prefix)]
+                       community-context-tag
+                       [quo/text {:style common-style/user-avatar-tag-text}
+                        (i18n/label :t/community-request-not-accepted-body-text-suffix)]]}
+        {:header-text (i18n/label :t/community-request-pending)
+         :context     [[quo/text {:style common-style/user-avatar-tag-text}
+                        (i18n/label :t/community-request-pending-body-text)]
+                       community-context-tag]})
+
+      (= membership-status constants/activity-center-membership-status-accepted)
+      {:header-text (i18n/label :t/community-request-accepted)
+       :context     [[quo/text {:style common-style/user-avatar-tag-text}
+                      (i18n/label :t/community-request-accepted-body-text)]
+                     community-context-tag]}
```

Comment:
I am not sure about this concatenation for translated strings. I think it may be problematic for some languages, specially RTL ones. It may work now for english, but problems may start to arise when we add new translations for this text.

Reply:
I agree with you, Brian. 

Good spot. I think RTL is something we haven't thought about during implementation/design.  

From design, the `activity-log` component has to render text with `context-tag(s)` in between. We have been using this component or concatenation of text and context tag(s) across all notifications in Activity Centre. A quick solution will be to reverse the concatenation if the locale is RTL. But, It is best to talk to the design team about the RTL behaviour of the Activity Center.

Reply:
Sure, I will create an issue to track this if does not already exists already.

Reply:
Issue here: https://github.com/status-im/status-mobile/issues/15709